rotkehlchen/data/all_assets.json

```diff
@@ -13451,6 +13451,66 @@
         "symbol": "sUSD",
         "type": "ethereum token"
     },
+    "sBTC": {
+        "coingecko": "sbtc",
+        "cryptocompare": "btc",
+        "ethereum_address": "0xfe18be6b3bd88a2d2a7f928d00292e7a9963cfc6",
```

Comment:
This and the other ethereum addresses also need to be checksummed when input in this json file. Can you fix that? This is why the tests fail.

Reply:
OK, I fixed that 👍